scripts/scrape.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import os
+import sys
+sys.path.append('src')  # if there is a better way, please tell me
+import wiktionary_scraper as ws
+import string_cleanup as sc
+import yaml
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--language', help='specify the language of the data used', required=True)
+    parser.add_argument('-i', help='path to input word list', required=True, dest='input_file')
+    parser.add_argument('-o', help='path to output file', required=True, dest='output_file')
+    args = parser.parse_args()
+
+    with open('scripts/script_config.yml', encoding='utf-8') as f:
```

Comment:
you can turn the config file into an argument of the scripts using argparse as you do. And you can use a default value in argparse to avoid having to type it each type. But i would say it's a good practice, this way  you can have different config files if needed (prod, dev, test, etc)

Reply:
done